packages/@okta/vuepress-site/docs/reference/token-hook/index.md

```diff
@@ -403,6 +557,92 @@ See [ID Token Claims](/docs/reference/api/oidc/#id-token-claims) for a list of I
 }
 ```
 
+You can also use the `remove` operation to remove members from existing JSON objects and elements from existing arrays. For example, you have an array that contains the user's preferred airports, and you want to remove an airport from the array.
+
+Existing target JSON object:
+```json
+{  
+   "preferred_airports":[  
+      "sjc",
+      "lax",
+      "sfo",
+      "oak"
+   ]
+}
+```
+You can remove the element from the array by specifying the array name followed by the index of the element that you want to remove. You don't need to specify a value for the `remove` operation. But, you can specify `null` as the value if you want. You can also remove a member from a JSON object by specifying the name of the object and the object member that you want to remove in the path. 
```

Comment:
seems like we don't need 
```
You can also remove a member from a JSON object by specifying the name of the object and the object member that you want to remove in the path.
```
since we added example with description for that case below

Reply:
good catch!